### DISCUSSION TITLE
How do you handle the increasing number of merge conflicts when using Tailwind?

### DISCUSSION THREAD

**Discussion by pascal08** [tailwindlabs/tailwindcss#8056] [Help]
Recently, we started using Tailwind on a project with around 25 collaborators. One thing we noticed is that the number of merge conflicts is increasing once we started using Tailwind because we have a larger change of conclicts on a single line. You could put every class on a new line to circumvent that but it's quite ugly. Another option is to use @apply but that's also taking away some benefits of using Tailwind inline classes. 

Main question: how do you prevent merge conflicts when collaborating on the styling of the same html elements?

**Comment by gedewon** [comment]
try  compiling  tailwind CSS at build time to avoid merge conflicts
